4/README.md

```diff
@@ -172,6 +172,32 @@ pointed by *__root_att* MUST be reflected in the forwarded Attribute.
 An Attribute MAY have associated dynamic metadata in form of
 Properties (see [RFC 5/Property](/5)).
 
+Below table summarizes summarizes all attribute properties.
+Detailed description is provided later in this section.
+
+| Property             | Name in database     | Type   | Default value                                                                                         |
+|----------------------|----------------------|--------|-------------------------------------------------------------------------------------------------------|
+| *description*        | `description`        | string | Not specified / No description                                                                        |
+| *label*              | `label`              | string | Not specified / No label                                                                              |
+| *unit*               | `unit`               | string | Not specified / No unit                                                                               |
```

Comment:
The default unit property C++ library default has been changed from "No Unit" to empty string in commit https://github.com/tango-controls/cppTango/commit/4aac0cc